l10n/de_DE.js

```diff
@@ -58,6 +58,14 @@ OC.L10N.register(
     "Redirect warning" : "Umleitungswarnung",
     "The previous page is sending you to %s" : "Die vorangegangene Seite leitet Sie zu %s weiter",
     "If you do not want to visit that page, you can return to <a href=\"%s\">the mail app</a>." : "Wenn Sie diese Seite nicht besuchen möchten, können Sie <a href=\"%s\">zur Mail-App</a> zurück kehren.",
-    "Continue to %s" : "Fortsetzen auf %s"
+    "Continue to %s" : "Fortsetzen auf %s",
+    "New message" : "Neue Nachricht",
+    "to" : "An",
+    "Subject" : "Betreff",
+    "Message …" : "Nachricht ...",
```

Comment:
Tip: you can type that three-dots-char with ``Alt Gr`` + ``.`` on German keyboards ;-)